README.md

```diff
@@ -7,7 +7,7 @@ description: 'The unofficial Discord.js beginner''s guide, written by idiots for
 ## Introduction
 
 {% hint style="info" %}
-This guide is for Discord.js v12. For v13, once it is released, please see the [Official Discordjs.Guide](https://discordjs.guide/), this version of the guide is **Work in Progress**, so some of the articles may not have been updated yet.
+This guide is for Discord.js v13.
```

Comment:
This entire info block could honestly be removed in the v13 update.

Reply:
Okay, will do just that :).